samples/bluetooth/hci_uart/src/main.c

```diff
@@ -400,5 +402,21 @@ void main(void)
 		if (err) {
 			LOG_ERR("Failed to send");
 		}
+
+#if defined(CONFIG_INIT_STACKS)
+		static u32_t prio_ts;
+
+		if (k_uptime_get_32() - prio_ts > K_SECONDS(5)) {
+			STACK_ANALYZE("main thread stack",
```

Comment:
STACK_ANALYZE is deprecated, please do not use it.
You want a k_thread_foreach() loop and call k_thread_stack_space_get() on each thread.

Reply:
Will replace STACK_ANALYZE usage. But need a way to do it for main stack. k_thread_foreach will not give me main stack.

Reply:
Replaced with suggestion.